src/components/Auth/Auth.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect } from 'react-router-dom';
+import { Form, Field } from 'react-final-form';
+
+class Auth extends Component {
+  componentDidMount() {
+    return (this.props.register.isLoggedIn ? <Redirect to="/home" /> : '');
```

Comment:
Can you use `Route`s to conditionally display your private pages instead of redirecting after the `component Has Mounted`?

Reply:
Done!